Backpack/src/androidTest/java/net/skyscanner/backpack/calendar/presenter/BpkCalendarControllerTest.kt

```diff
@@ -127,6 +127,19 @@ class BpkCalendarControllerTest {
     verify(spy, times(3)).onRangeSelected(expectedRange)
   }
 
+  @Test
+  fun test_isToday() {
+    val today = Calendar.getInstance()
+    val day = today.get(Calendar.DAY_OF_MONTH)
+    val month = today.get(Calendar.MONTH)
+    val year = today.get(Calendar.YEAR)
+
+    Assert.assertTrue(subject.isToday(year, month, day))
+    Assert.assertFalse(subject.isToday(year, month, day + 1))
+    Assert.assertFalse(subject.isToday(year, month + 1, day))
+    Assert.assertFalse(subject.isToday(year + 1, month, day))
```

Comment:
Why are these three `assertFalse` assertions needed? Usually it's more than enough to test the correct expected assertion and not over-specify it with alternative ones.

Reply:
Because with only the first one the method could be just returning true and it would pass. Those assertions is to make sure all parameters are being used correctly.

Reply:
Gotcha. Makes sense now if I think about it. 👍